code/game/machinery/cloning.dm

```diff
@@ -309,23 +306,13 @@
 		mess = 0
 		gibs(loc)
 		icon_state = "pod_0"
-
-		/*
-		for(var/obj/O in src)
-			O.loc = src.loc
-		*/
 		return
 
 	if (!occupant)
 		return
-	/*
-	for(var/obj/O in src)
-		O.loc = src.loc
-	*/
-
-	if(occupant.loc == src)

```

Comment:
The only way for a clonee to leave the pod without clearing the occupant var is through badminery.